webapp/src/components/create_legal_hold_form.tsx

```diff
@@ -152,15 +153,19 @@ const CreateLegalHoldForm = (props: CreateLegalHoldFormProps) => {
                     >
                         <input
                             type='checkbox'
-                            id='legal-hold-exclude-public-channels'
-                            checked={excludePublicChannels}
-                            onChange={excludePublicChannelsChanged}
+                            id='legal-hold-include-public-channels'
+                            checked={includePublicChannels}
+                            onChange={includePublicChannelsChanged}
                             className={'create-legal-hold-checkbox'}
                         />
-                        <label htmlFor={'legal-hold-exclude-public-channels'}>
-                            {'Exclude public channels'}
+                        <label htmlFor={'legal-hold-include-public-channels'}>
+                            {'Include public channels'}
                         </label>
                     </div>
+                    <div style={{display: (includePublicChannels) ? 'block' : 'none'}}>
+                        <i className='icon icon-alert-outline'/>
+                        <span>{'It is possible for users to access public content without becoming members of a public channel. This setting only captures public channels that users are members of.'}</span>
```

Comment:
@cwarnermm can we capture this in the documentation as well?

Reply:
https://github.com/mattermost/docs/pull/7411 updated accordingly